src/getting-started/interacting/how_to_deploy.md

```diff
@@ -0,0 +1,52 @@
+## Declaring and Deploying the SimpleStorage contract
```

Comment:
@Keep it generic, Declaring and Deploying a contract. And just before mention "we will use this contract ..." linking to the starknet by example relevant chapter with the simple contract (such as StorageVariablesExample)